manifests/init.pp

```diff
@@ -2,18 +2,23 @@
 # == Description: Installs the go-carbon package and the shared configuration files
 #
 class go_carbon(
-  $ensure                   = 'present',
-  $package_name             = $go_carbon::params::package_name,
-  $version                  = $go_carbon::params::version,
-  $executable               = $go_carbon::params::executable,
-  $systemd_service_folder   = $go_carbon::params::systemd_service_folder,
-  $config_dir               = $go_carbon::params::config_dir,
-  $user                     = $go_carbon::params::user,
-  $group                    = $go_carbon::params::group,
-  $storage_aggregations     = $go_carbon::params::storage_aggregations,
-  $storage_schemas          = $go_carbon::params::storage_schemas,
-  $download_package         = $go_carbon::params::download_package,
-  $shell                    = $go_carbon::params::shell,
+  $ensure                         = 'present',
+  $package_name                   = $go_carbon::params::package_name,
+  $version                        = $go_carbon::params::version,
+  $download_deb_url               = "https://github.com/lomik/go-carbon/releases/download/v${version}/go-carbon_${version}_amd64.deb",
+  $executable                     = $go_carbon::params::executable,
+  $systemd_service_folder         = $go_carbon::params::systemd_service_folder,
+  $config_dir                     = $go_carbon::params::config_dir,
+  $user                           = $go_carbon::params::user,
+  $group                          = $go_carbon::params::group,
+  $storage_aggregations           = $go_carbon::params::storage_aggregations,
+  $storage_schemas                = $go_carbon::params::storage_schemas,
+  $download_package               = $go_carbon::params::download_package,
+  $shell                          = $go_carbon::params::shell,
+  $max_cpu                        = $go_carbon::params::max_cpu,
+  $whisper_workers                = $go_carbon::params::whisper_workers,
+  $whisper_max_updates_per_second = $gocarbon::params::whisper_max_updates_per_second,
+  $whisper_max_creates_per_second = $gocarbon::params::whisper_max_creates_per_second	
```

Comment:
Guessing this is a type `$gocarbon` vs `$go_carbon`

Reply:
Typo in my typo.

Reply:
just a typo..